openshift/hack/generate_requirements.py

```
@@ -0,0 +1,1259 @@
+
+    for phase_key, txt_path, in_path in build_phases:
+        label = txt_path.stem.replace("requirements-", "")
+        print(f"\n  Scanning {txt_path.name}…", flush=True)
+
+        # safety lives inside the pipenv virtualenv; try several invocation forms
+        for safety_cmd in (
+            ["pipenv", "run", "safety", "check", "-r", str(txt_path)],
+            ["python3", "-m", "safety", "check", "-r", str(txt_path)],
+            ["safety", "check", "-r", str(txt_path)],
+        ):
+            check = subprocess.run(safety_cmd, capture_output=True, text=True)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)